src/redux/categories/categories.js

```diff
@@ -0,0 +1,30 @@
+export const categoriesState = [];
+
+const ADD_CATEGORY = 'ADD_CATEGORY';
+const REMOVE_CATEGORY = 'REMOVE_CATEGORY';
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] **[OPTIONAL]** The value of  action type needs to follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:   `const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'`